### ISSUE TITLE
$.selectOptionByValue() reports a misleading error message

### ISSUE THREAD

**Issue by asolntsev** [codeborne/selenide#709]
Steps to reproduce:
1. Given a page with `<select>` and options `val1` and `val2`.
2. Call method `$("select").selectOptionByValue("val3");`.
3. Selenide shows a misleading error message:

```
Element not found {By.cssSelector: select}
Expected: exist
Caused by: NoSuchElementException: Cannot locate option with value: val3
```

Expected result: Selenide should show a correct error message like "Option with value 'val3' is not found".

## environment
* **Selenide Version: 4.10.01**

**Comment by BorisOsipov** [comment]
@asolntsev are you sure expected results can be fulfilled? as I know it is hard to implement(

**Comment by asolntsev** [comment]
@BorisOsipov I don't know yet, I haven't researched it yet.

**Comment by rosolko** [comment]
@asolntsev @BorisOsipov We can trick a little `SelectOptionByValue` method and get error message like:
```
Element not found {[@livemail.ru, @myrambler.ru, @rusmail.ru, @мыло.ру]}
Expected: value 'myrambler.ru1'
```

**Comment by rosolko** [comment]
@asolntsev @BorisOsipov with little tuning of `selectOptionByValue` we can get error like:
```
Element not found {values: [@livemail.ru, @myrambler.ru, @rusmail.ru, @мыло.ру]}
Expected: value 'myrambler.ru1'
```

**Comment by asolntsev** [comment]
Fixed in https://github.com/codeborne/selenide/pull/780, will be released in Selenide 4.12.4
